README.md

```diff
@@ -11,10 +11,13 @@ It takes a few steps to setup flexDATA on your machine. We recommend that the us
 
 Simply install with:
 ```
-conda create -n <your-environment> python=3.6
+conda create -n <your-environment> python=3.7
 conda install -c cicwi -c astra-toolbox/label/dev -c conda-forge -c owlas flexdata
 ```
 
+**NOTE**: currently, the latest version of flexdata is only available
+for python version 3.7.
```

Comment:
**NOTE**: currently, the latest version of flexDATA is only available
for Python version 3.7.

Reply:
Capitalization issue?